txbits/app/wallet/Wallet.scala

```diff
@@ -304,6 +358,7 @@ class Wallet(rpc: JsonRpcHttpClient, currency: CryptoCurrency, nodeId: Int, para
   }
 
   private def getBalance = {
+    // This method is unused as balance is now tracked in the database
```

Comment:
We can wrap the sendmany inside of a try/catch if we want to handle that case differently. We do not update the balance of a hot wallet directly, only through functions for deposits/withdrawals and cold storage. I don't think changing the balance is the right solution.